### ISSUE TITLE
Deleting messages

### ISSUE THREAD

**Issue by madsmtm** [carpedm20/fbchat#376]
Facebook recently added the ability to delete messages. This was implemented as `unsending` by @kapi2289 in #333, but that might be broken now?

I just looked into it, seems like message deletion moved to `https://www.facebook.com/ajax/mercury/delete_messages.php`, which you POST with:
```py
"message_ids[0]": "mid.XYZ"
```
Plus all the other usual Facebook cruft.

This suggests that we may be able to delete multiple messages at once? Don't have the time to investigate further right now.

**Comment by kapi2289** [comment]
`unsend` still works for me, maybe you have new functions that are not yet in my country.

**Comment by kapi2289** [comment]
Okay, now I know. You (or your bot) haven't got unsending function yet. This URL you provided is for private deletion that is already in `fbchat`, added in #318

**Comment by madsmtm** [comment]
Could very probably be. I can't detect these with the `deleted` tag on `Message` either, though I can detect it with a different `delta` class, `"MessageDelete"`:
```json
{
  "delta": {
    "class": "MessageDelete",
    "irisSeqId": "1234567",
    "messageIds": [
      <message_id>
    ],
    "randomNonce": 123456789,
    "requestContext": {
      "apiArgs": {}
    },
    "threadKey": {
      "threadFbId": <thread_id>
    }
  },
  "iseq": 1234567,
  "ofd_ts": <timestamp>,
  "queue": <user_id>,
  "type": "delta"
}
```

**Comment by kapi2289** [comment]
For handling this is `onMessageDeleted` 😉

**Comment by madsmtm** [comment]
Yeah, but there's a difference between deleted messages, and unsent messages 🤔
